build/automation/var/profile/demo.mk

```diff
@@ -5,20 +5,21 @@
 
 LOG_LEVEL:= INFO
 
-DOS_API_GATEWAY_SECRETS = core-dos-db-sync/deployment
-DOS_API_GATEWAY_USERNAME_KEY := DOS_API_GATEWAY_USER
-DOS_API_GATEWAY_PASSWORD_KEY := DOS_API_GATEWAY_PASSWORD
-DOS_API_GATEWAY_REQUEST_TIMEOUT := 30
-DOS_API_GATEWAY_URL := https://core-dos-performance-ddc-core-dos-api-gateway.k8s-nonprod.texasplatform.uk/api/change-request
+DOS_API_GATEWAY_SECRETS = core-dos-uet/deployment
+DOS_API_GATEWAY_USERNAME_KEY := DOS_UET_API_GATEWAY_USER
+DOS_API_GATEWAY_PASSWORD_KEY := DOS_UET_API_GATEWAY_PASSWORD
+DOS_API_GATEWAY_URL := https://core-dos-put-ut-ddc-core-dos-api-gateway.k8s-prod.texasplatform.uk/api/change-request
+
+
 MOCK_MODE := false
 
-DB_SERVER_NAME := uec-core-dos-performance-db-12-replica-di
+DB_SERVER_NAME := uec-core-dos-put-db-12-replica-di
```

Comment:
is this value should be uec-core-dos-**uet**-db-12-replica-di or uec-core-dos-put-db-12-replica-di

Reply:
is correct as is